packages/contracts/contracts/L1/sequencer/Sequencer.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.9.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
+
+contract Sequencer is ReentrancyGuardUpgradeable, OwnableUpgradeable {
```

Comment:
Why must extend `ReentrancyGuardUpgradeable`? There are no interacting methods.

Reply:
Methods that need to call other contracts(ERC20) should extend `ReentrancyGuardUpgradeable`